hatch_jupyter_builder/migrate/_migrate.py

```diff
@@ -6,10 +6,14 @@
 import sys
 from pathlib import Path
 
-import tomli
 import tomli_w
 from packaging import version
 
+try:
+    import tomllib
+except ImportError:
+    import tomli as tomllib
```

Comment:
This should make the type checker happy. Type checkers don't understand `try/except` clauses for import statements very well, but they're much happier with `sys.version_info` guards.

```suggestion
if sys.version_info >= (3, 11):
    import tomllib
else:
    import tomli as tomllib
```